go.mod

```diff
@@ -8,11 +8,11 @@ require (
 	cosmossdk.io/math v1.0.0-beta.3
 	github.com/99designs/keyring v1.2.1 // indirect
 	github.com/BurntSushi/toml v1.2.0
-	github.com/celestiaorg/celestia-app v0.7.0-rc-1
+	github.com/celestiaorg/celestia-app v0.7.0-rc5.0.20221013001953-3d648a95918f
```

Comment:
[for reviewers] we should probably merge https://github.com/celestiaorg/celestia-app/pull/866 cut v0.7.0-rc6 and upgrade to that here before merging this PR